File: SenorPotato/Player.fs
module Player

type Skills = { Defence: int
                Attack: int
                Penalty: int
                Stamina: int }

type Player = { Name: string
                Description: string
                Overall: int
                Skills: Skills
                CaptainBonus: Skills->Skills
                IsCaptain: bool }
                with member this.CompleteSkills = if this.IsCaptain then this.CaptainBonus this.Skills else this.Skills

File: Game/Player.cs
namespace Game
{
    public class Player
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Country { get; set; }
        public int Overall { get; set; }
        public Skills Skills { get; set; }
        public SpecialSkill SpecialSkill { get; set; }
        public bool IsCaptain { get; set; }
    }
}